src/cmd/list.go

```diff
@@ -121,35 +129,22 @@ func list(cmd *cobra.Command, args []string) error {
 }
 
 func getContainers() ([]toolboxContainer, error) {
-	logrus.Debug("Fetching containers with label=com.github.containers.toolbox=true")
-	args := []string{"--all", "--filter", "label=com.github.containers.toolbox=true"}
-	containers_old, err := podman.GetContainers(args...)
-	if err != nil {
-		return nil, fmt.Errorf("failed to list containers with label=com.github.containers.toolbox=true: %w", err)
-	}
+	var err error
+	var args []string
```

Comment:
Not all of these variables need to be defined separately.

Reply:
You're right, I only did it to make it more readable. I guess that's the C programmer in me talking :).

Reply:
See https://github.com/containers/toolbox/pull/799